fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -46,10 +46,9 @@ def self.cruise_lane(platform, lane, parameters = nil, env = nil, fastfile_path
       # and not return to the original working directory
       # https://github.com/CocoaPods/Xcodeproj/issues/426
       # Setting this environment variable causes xcodeproj to work around the problem
+      # TODO: Does this need moving into CLIToolsDistributor to be called **before** loading Dotenv?
       ENV["FORK_XCODE_WRITING"] = "true" unless platform == 'android'
```

Comment:
I _think_ its safe for us to remove this now..

I traced it back to #6278, which is also discussed in https://github.com/CocoaPods/Xcodeproj/issues/426#issuecomment-251804075 where they talk about it being resolved in 1.4.0 of Xcodeproj...

If that's the case, since the gemspec is currently requires >= 1.13.0 meaning that the bug is no longer a thing... 

https://github.com/fastlane/fastlane/blob/204fab2f681dad3a60314844e8e062b19cc1bf28/fastlane.gemspec#L60

WDYT?

Reply:
Seems safe to remove!